src/views/README.md

```diff
@@ -25,3 +25,61 @@ E.G. the Tree *view* displays Cylc cycles/tasks/jobs in a collapsible hierarchy.
 It uses the Tree *component* which provides the generic tree logic e.g.
 indentation, expand/collapse, etc. This Tree *component* is also used by
 the Workflows *view*.
+
+
+## Subscriptions
```

Comment:
Added some details here as the rules for subscription writing are getting a little complex.

See also https://github.com/cylc/cylc-ui/issues/862 for context, and a possible way to simplify things for view writers.